src/main/java/teammates/common/datatransfer/attributes/FeedbackSessionAttributes.java

```diff
@@ -427,6 +429,41 @@ public void setInstructions(Text instructions) {
         this.instructions = instructions;
     }
 
+    public String getCreatedTimeDateString() {
+        return TimeHelper.formatDateForInstructorCoursesPage(createdTime, timeZone);
```

Comment:
Oh I just follows the format of naming in `CourseAttributes.java`. But the attribute name there is `createdAt` ->`getCreatedAtDateString()`... Yeah I agree that the name here is a bit confusing. Got any suggestions? Or shall we just leave it?

Reply:
@darrenwee what do you think?

Reply:
@darrenwee what do you think?

Reply:
How about `getCreatedDateStamp` since you're using `formatDate...`?

Reply:
Then how should we name the method below it (`getCreatedTimeDateStamp()`)?

My suggestions for these three method names:
`getCreatedDateString()`
`getCreatedDateTimeStamp()`
`getCreatedDateTimeFullString()`

@darrenwee @joanneong What do yall think?

Reply:
Oh oops I relooked what I wrote, and actually my concern was with `formatDateForInstructorCoursesPage` because now you're formatting the date for instructor sessions page...sorry for the confusion :(

Reply:
Oops sorry...misunderstanding :(

Reply:
I think I should add a new method in `TimeHelper.java`